tests/Makefile

```diff
@@ -15,13 +15,23 @@ test-ci:
 	nohup kubectl port-forward -n nvsentinel svc/simple-health-client 8080:8080 > /dev/null 2>&1 &
 	kubectl get po -Ao wide
 	kubectl get no -o wide
+	@echo "Pre-test: Ensuring MongoDB initialization is complete..."
+	kubectl wait --for=condition=complete job/create-mongodb-database -n nvsentinel --timeout=5m || true
```

Comment:
Can we add the job status check to https://github.com/NVIDIA/NVSentinel/blob/main/Makefile#L669-L684 instead? Also if you do `|| true` doesn't it mean that it will pass even if it fails?